lib/code_climate/test_reporter.rb

```diff
@@ -1,16 +1,5 @@
 module CodeClimate
   module TestReporter
-    def self.start
```

Comment:
Maybe we should keep this around but have it output a deprecated log message & exit? I'm sure there will be users who upgrade the gem in their gemfile at some point & wonder why they're not getting results.